lib/v2/guangdiu/search.js

```diff
@@ -0,0 +1,40 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseRelativeDate } = require('@/utils/parse-date');
+
+const host = 'https://guangdiu.com';
+
+module.exports = async (ctx) => {
+    const query = ctx.params.query ?? '';
+    const url = `${host}/${query ? `search.php?${query}` : ''}`;
+    const response = await got(url);
+    const $ = cheerio.load(response.data);
+    const list = $('#mainleft > div.zkcontent > div.gooditem')
+        .map((_index, item) => ({
+            title: $(item).find('a.goodname').text().trim(),
+            link: new URL($(item).find('div.iteminfoarea > h2 > a').attr('href'), host).href,
```

Comment:
Please use the `href` from `a.goodname` instead

Reply:
done~